client/src/components/pages/NeowisePage.js

```diff
@@ -1,22 +1,24 @@
-import { OrbitControls } from "drei";
+import { HTML, OrbitControls, Stars } from "drei";
 import React, { Suspense } from "react";
 import { Canvas } from "react-three-fiber";
-import styled from "styled-components";
 import SpaceShip from "../3D/SpaceShip";
-const CanvasStyled = styled(Canvas)`
-  height: 100vh;
-`;
+import Input from "../Input/Input";
 
 function NeowisePage() {
   return (
-    <CanvasStyled colorManagement camera={{ position: [-5, 2, 10], fov: 70 }}>
+    <Canvas colorManagement camera={{ position: [-5, 2, 10], fov: 70 }}>
       <directionalLight position={[0, 10, 0]} intensity={0.5} />
       <Suspense fallback={null}>
         <SpaceShip />
       </Suspense>
-
+      <HTML prepend position={[7, 1, 1]}>
+        <Input placeholder="Type a keyword like Earth"></Input>
+      </HTML>
```

Comment:
isnt it a bit confusing, the naming of a component with html? its usually the surrounding tag for the hole site, could you specifiy the component

Reply:
It´s not a common HTML. It´s HTML coming from "drei" to combine a 3D surrounding with 2D HTML Elements.

Reply:
And thanks for review! 😊